readme.md

```diff
@@ -50,7 +50,7 @@ This repository is a monorepo, and contains the following packages maintained wi
 
 ### Installing dependencies
   * **yarn** - version 1.22.4, for easy management of specific Yarn versions, we recommend using [Yarn Version Manager (YVM)](https://github.com/tophat/yvm).
-  * **node** - version 2.18.0, for easy management of specific versions, we recommend using [n](https://github.com/tj/n).
+  * **node** - version 12.18.0, for easy management of specific versions, we recommend using [n](https://github.com/tj/n).
```

Comment:
```suggestion
  * **node** - version 12.16.3, for easy management of specific versions, we recommend using [n](https://github.com/tj/n).
```

We've got `12.16.3` in our `.nvmrc`, and specify it to be used by some github actions. So, I think we should stick to that in the readme!